### PR TITLE
change type of noRecentsText from String to Widget

### PR DESCRIPTION
**Pull request by LostInDarkMath**
Hi there!
Thanks for your awesome package! I'll suggest a small change because if missed a feature: It's not possible to use a custom padding for the `noRecentsText`. To allow more customization of the `noRecentsText`, I changed to type from `String` to `Widget`.
This also allows to use custom imgaes instead of  a text :)

### CODE REVIEW

File: lib/src/config.dart [hunk 1]
```diff
@@ -81,11 +83,8 @@ class Config {
   /// Limit of recently used emoji that will be saved
   final int recentsLimit;
 
-  /// The text to be displayed if no recent emojis to display
-  final String noRecentsText;
-
-  /// The text style for [noRecentsText]
-  final TextStyle noRecentsStyle;
+  /// A widget (usually [Text]) to be displayed if no recent emojis to display
+  final Widget noRecentsText;
```
Comment (Fintasys) [inline lib/src/config.dart:87]: What do you think about renaming the attribute to `noRecentsWidget` to make it more consistent with the type? Or is the meaning more obvious by calling it "text" ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.2.0
+* New feature & breaking change: `noRecentsText` is now of type `Widget` (was `String` before). 
+  Inconsequence, `noRecentsStyle` was removed. This change leads to more customization options.
```
Comment (Fintasys) [inline CHANGELOG.md:3]: Just small suggestion to match existing format
```suggestion
- Breaking Change: `noRecentsText` is now of type `Widget` (was `String` before) and `noRecentsStyle` was removed from Config
```
Eventually need to be adjusted if property is renamed.

**Review by Fintasys — COMMENTED** [review]
Thank you very much for your contribution. I support your idea and I have a few small comments on your code. Please have a look 🙇

File: lib/src/config.dart (hunk 1 above)
Comment (LostInDarkMath) [reply lib/src/config.dart:87]: Good question. I'm not too sure about this. Maybe we should name it just `noRecents`. I think it's bad code style to include the type in the name. What do you think?

File: lib/src/config.dart [hunk 3]
```diff
@@ -24,9 +24,11 @@ class Config {
...
       this.showRecentsTab = true,
       this.recentsLimit = 28,
-      this.noRecentsText = 'No Recents',
-      this.noRecentsStyle =
-          const TextStyle(fontSize: 20, color: Colors.black26),
+      this.noRecentsText = const Text(
+        'No Recents',
+        style: TextStyle(fontSize: 20, color: Colors.black26),
+        textAlign: TextAlign.center,
+      ),
```
[the remark this answers is not in the document]
Comment (LostInDarkMath) [reply lib/src/config.dart:31]: Sure, I can live with both :) So I'll apply your change if the naming of `noRecentsText` is clear (see comment above).

File: lib/src/config.dart (hunk 1 above)
Comment (Fintasys) [reply lib/src/config.dart:87]: Right, sounds good to me ! 👍

**Comment by LostInDarkMath** [comment]
@Fintasys I applied all of your suggested changes in https://github.com/Fintasys/emoji_picker_flutter/pull/61/commits/b3944636d02be976708ae729627918070f368acf
Let me know what you think about it :)

**Review by Fintasys — APPROVED** [review]
LGTM ! Danke für die Verbesserung! 😃

**Comment by LostInDarkMath** [comment]
Gerne doch :)

**Pull request merged**